src/UIModule.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace RebelCode\Bookings\Wordpress\Module;
+
+use Dhii\Data\Container\ContainerFactoryInterface;
+use Psr\Container\ContainerInterface;
+use Psr\EventManager\EventManagerInterface;
+use RebelCode\Modular\Module\AbstractBaseModule;
+
+class UIModule extends AbstractBaseModule
+{
+    /**
+     * Constructor.
+     *
+     * @since [*next-version*]
+     *
+     * @param ContainerFactoryInterface $containerFactory The factory for creating container instances.
+     * @throws \Dhii\Exception\InternalException
+     */
+    public function __construct(ContainerFactoryInterface $containerFactory)
+    {
+        $this->_initModule(
+            $containerFactory,
+            'bookings-ui-module',
+            ['wp-events'],
+            $this->_loadPhpConfigFile(__DIR__ . '/../config.php')
+        );
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function setup()
+    {
+        return $this->_createContainer([
+            'template-manager' => function (ContainerInterface $c) {
+                $templateManager = new TemplateManager($c);
+                $templateManager->registerTemplates();
+                return $templateManager;
+            }
+        ]);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function run(ContainerInterface $c = null)
+    {
+        /* @var $eventManager EventManagerInterface */
+        $eventManager = $c->get('event-manager');

```

Comment:
As per our recent agreement, here and wherever else necessary, change the service keys declared to `snake_case`. Also assume that services provided by other modules are also in `snake_case`.